lxd/storage_dir.go

```diff
@@ -773,6 +793,35 @@ func (s *storageDir) ContainerRename(container container, newName string) error
 		}
 	}
 
+	// Rename the backup mountpoint for the container if existing:
+	// ${POOL}/backups/<old_container_name> to ${POOL}/backups/<new_container_name>
+	oldBackupsMntPoint := getBackupMountPoint(s.pool.Name, container.Name())
+	newBackupsMntPoint := getBackupMountPoint(s.pool.Name, newName)
+	if shared.PathExists(oldBackupsMntPoint) {
+		err = os.Rename(oldBackupsMntPoint, newBackupsMntPoint)
+		if err != nil {
+			return err
+		}
+	}
+
+	// Remove the old backup symlink:
+	// ${LXD_DIR}/backups/<old_container_name>
+	oldBackupSymlink := shared.VarPath("backups", container.Name())
+	newBackupSymlink := shared.VarPath("backups", newName)
+	if shared.PathExists(oldBackupSymlink) {
+		err := os.Remove(oldBackupSymlink)
+		if err != nil {
+			return err
+		}
+
+		// Create the new backup symlink:
+		// ${LXD_DIR}/backups/<new_container_name> -> ${POOL}/backups/<new_container_name>
+		err = os.Symlink(newBackupsMntPoint, newBackupSymlink)
```

Comment:
These should all go away.

Reply:
@brauner can you push a commit to take care of those as you have more background on what's that about than I do?